contracts/truefi/TrueFiPool.sol

```diff
@@ -256,6 +287,14 @@ contract TrueFiPool is ITrueFiPool, ERC20, ReentrancyGuard, Ownable {
         return _stakeToken.balanceOf(address(this));
     }
 
+    /**
+     * @dev Get total balance of stake tokens
+     * @return Balance of stake tokens in this contract
+     */
+    function crvBalance() public view returns (uint256) {
+        return _minter.token().balanceOf(address(this));
```

Comment:
does this include unclaimed crv balance? should we be including that when calculating the total pool value?

Reply:
No it doesn't and I think that including it would be wrong. These tokens are not in the pool yet and it may lead to some unexpected problems if we include it into the value calculation.